src/navigation/components/NavigationItem/components/ContentWrapper/ContentWrapper.scss

```diff
@@ -7,7 +7,7 @@ $block: '.#{$ns}content-wrapper';
         display: flex;
         width: 20px;
         height: 20px;
```

Comment:
can we delete this styles with default iconSize?

Reply:
of course